app/components/AllOrders.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import { connect } from 'react-redux'
+
+export class AllOrders extends React.Component {
+  render () {
+    return (
+      <div>
+        <ul className="list-unstyled">
+          {
+            this.props.allOrders.map(order => {
+              return (
+                <div key={order.id}>
```

Comment:
not 100% sure on this, but i think the `.map()` needs to return `<li>` elements. these function is wrapped in a `<ul>` element.